docs/sql-ref-datatypes.md

```diff
@@ -193,7 +193,7 @@ The following table shows the type names as well as aliases used in Spark SQL pa
 |**BinaryType**|BINARY|
 |**DecimalType**|DECIMAL, DEC, NUMERIC|
 |**CalendarIntervalType**|INTERVAL|
-|**ArrayType**|ARRAY<element_type>|
+|**ArrayType**|ARRAY\<element_type>|
```

Comment:
Interesting .. why do we have to escape this alone?
Looks good anyway as it works ..

Reply:
Oh, it thinks it's an XML.

Reply:
I am also curious about why `STRUCT` and `Map` do not require escape.

Reply:
This behavior comes from [kramdown](https://kramdown.gettalong.org/) which is used by Jekyll.

The `element_type` is interpreted as an html tag because it consists of only alphabet characters.
On the other hand, `field1_name: field1_type, field2_name: field2_type, ...`(STRUCT) and `key_type, value_type`(MAP) contain bare comma, so they cannot be interpreted as html tags.

Strictly speaking, kramdown uses the following regular expression to determine whether it is an html tag or not.

```
(?m-ix:<((?>(?:[[:alpha:]_][-[:alnum:]._]*:)?[[:alpha:]_][-[:alnum:]._]*))\\s*((?>\\s+(?:[[:alpha:]_][-[:alnum:]._]*:)?[[:alpha:]_][-[:alnum:]._]*(?:\\s*=\\s*(?:\\p{Word}+|(\"|').*?\\3))?)*)\\s*(\\/)?>)
```